airflow/example_dags/example_xcom.py

```diff
@@ -43,24 +44,14 @@ def push_by_returning(**kwargs):
     return value_2
 
 
-def puller(**kwargs):
+def puller(pulled_value_1, pulled_value_2, **kwargs):
```

Comment:
```suggestion
def puller(pulled_value_1, pulled_value_2):
```
We can probably drop the `kwargs`

Reply:
I think I need that for getting the ti as we want to put an example using the old syntax and comparing it to new one like you mentioned in your previous comment.